src/main/java/com/shootoff/camera/cameratypes/PS3EyeCamera.java

```diff
@@ -77,18 +111,28 @@ public static void init() {
 		if (initialized)
 			return;
 
+		try {
+
+			eyecamLib = (eyecam) Native.loadLibrary("eyecam", eyecam.class);
+
+		} catch (UnsatisfiedLinkError exception) {
+
+			initialized = false;
+		}
+
 		eyecamLib.ps3eye_init();
```

Comment:
Two choices here.  I either put a return at the end of the catch block, or I put all of the init code in the try block.  Which is better?